modules/end-user-guide/pages/mounting-configmaps-and-secrets.adoc

```diff
@@ -0,0 +1,106 @@
+:navtitle: Mounting ConfigMaps and Secrets
+:keywords: user-guide, configuring, user, configmaps, configmap, secret, secrets, mounting, mount
+:page-aliases:
+
+[id="mounting-configmaps-and-secrets_{context}"]
+= Mounting ConfigMaps and Secrets
+
+ConfigMaps are {platforms-name} objects that store non-confidential data such as configuration values for an application.
+
+Secrets are {platforms-name} objects that store confidential data such as user names, passwords, authentication tokens, and configurations in an encrypted form.
+
+This section describes how to automatically mount {orch-name} ConfigMaps and Secrets to all {devworkspace} containers, creating permanent mount points for components such as:
+
+* Maven configuration, such as the `settings.xml` file 
+* SSH key pairs
+* AWS authorization tokens
+* Configuration files
+* Git credentials store files
```

Comment:
Nice job here because you are setting the context, explaining why a user will want to inject secret or configmap. 👍

Reply:
This is from the original docs :)   https://www.eclipse.org/che/docs/stable/end-user-guide/configuring-user-secrets/